warehouse/utils/project.py

```diff
@@ -102,20 +104,27 @@ def quarantine_project(project: Project, request, flash=True) -> None:
     """
     Quarantine a project. Reversible action.
     """
+    # TODO: This should probably be extracted to somewhere more general for tasks,
+    #  but it got confusing where to add it in the context of this PR.
+    #  Since JournalEntry has FK to `User`, it needs to be a real object.
+    actor = request.user or request.db.scalar(
+        select(User).where(User.username == "admin")
+    )
```

Comment:
We do have `get_admin_user` [here](https://github.com/pypi/warehouse/blob/c477e43c7b7910acab6f5d39cdd70589e15e85d0/warehouse/accounts/services.py#L138-L141), although you'd need to grab a `IUserService` to use it.

Reply:
Thanks, I used that instead